ci/playbooks/kuttl/run-kuttl-tests.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: 'Get resource status before {{ operator }}_kuttl run'
+  ansible.builtin.shell: |
+    oc get {{ item }} > {{ cifmw_artifacts_basedir }}/logs/{{ item }}_before_{{ operator }}_kuttl.log
+  loop:
+    - pv
+    - all
+
```

Comment:
suggestion as per discussion - add a generalized command - that can be added via variable in a job.
Loop through a list command - before and after